bgpd/bgp_updgrp.c

```diff
@@ -497,6 +506,11 @@ static unsigned int updgrp_hash_key_make(const void *p)
 				   PEER_CAP_ORF_PREFIX_SM_OLD_RCV),
 			(intmax_t)CHECK_FLAG(peer->af_flags[afi][safi],
 					     PEER_FLAG_MAX_PREFIX_OUT));
+
+		zlog_debug(
+			"%pBP Update Group Hash: RTC: %u", peer, peer->afc[AFI_IP][SAFI_RTC]
+		);
+
```

Comment:
debug bgp update-groups?

Reply:
actually, I realise the zlog_debug() call was in a debug guard call. so it is ok too.